jira/src/main/kotlin/gropius/sync/jira/model/IssueData.kt

```diff
@@ -52,10 +53,27 @@ class JiraTimelineItem(val id: String, val created: String, val author: JsonObje
         return id
     }
 
+    fun isStringAndNullTemplate(encodedTypes: String): Pair<Boolean, Boolean> {
```

Comment:
Imho I think to separate functions would have been better

Reply:
1bc373190e3a64750877f6f1630f89afd6902635

Reply:
as nullable depends on the difference between any and string, it would require duplicating the whole function.

Reply:
fa99ad45520a153b257aca5f0d0ba0914308f9e6